src/main/java/com/madgag/gif/fmsware/AnimatedGifEncoder.java

```diff
@@ -345,6 +372,30 @@ protected int findClosest(Color c) {
 	}
 	
 	/**
+	 * Returns index of palette color closest to c
+	 *
+	 */
+	protected int findExact (Color c) {
+		if (colorTab == null) return -1;
+		int r = c.getRed() & 0xff;
+		int g = c.getGreen() & 0xff;
+		int b = c.getBlue() & 0xff;

```

Comment:
Why do these values need to be `& 0xff` - when the equivalent part in [`findClosest()`](https://github.com/junquero/animated-gif-lib-for-java/blob/dedd30da/src/main/java/com/madgag/gif/fmsware/AnimatedGifEncoder.java#L353-L355) currently doesn't?